pipelines/README.md

```diff
@@ -2,7 +2,11 @@
 
 ## Models
 
-This repository contains the following trained example MLflow models: [bike-rentals-auto-ml](models/bike-rentals-auto-ml/) and [tensorflow-housing](models/tensorflow-housing/):
+This repository contains the following trained examples.
+
+### MLflow models
+
+The [bike-rentals-auto-ml](models/bike-rentals-auto-ml/) and [tensorflow-housing](models/tensorflow-housing/):
```

Comment:
It is probably better to have a list of models and in the description tell which runtimes can run the models.

From the existent ones (@piotrpdev may confirm):
* ´bike-rentals-auto-ml´ is using MLFlow format and can run in Seldon.
* ´tensorflow-housing´ is using MLFlow format and wraps a TensorFlow model. It can run in Seldon, but can also run in OVMS by loading the `tf2model/` artifacts.

For the new ones, what I can infer is that:
* `onnx-mnist` is in ONNX format that can run on OVMS.
* `tensorflow-facedetection` is using OpenVino IR propietary format and would run only on OVMS.
* Both `lightgbm-iris` and `lightgbm-mushrooms` seem to be using Booster format which can run on Seldon.